ui/vite.config.ts

```diff
@@ -31,20 +31,30 @@ export default defineConfig(({ mode, command }) => {
     esbuild: {
       pure: ["console.debug"],
     },
-    build: { outDir: isCloud ? "dist" : "../static" },
+    build: {
+      outDir: isCloud ? "dist" : "../static",
+      rollupOptions: {
+        external: ["tesseract.js"],
+        output: {
+          paths: {
+            "tesseract.js": "https://cdn.jsdelivr.net/npm/tesseract.js@6.0.1/dist/tesseract.esm.min.js",
```

Comment:
Not loving the version hardcoding here... pretty buried. I wonder if _actually adding_ the tesseract NPM module to package.json and the exracting the version number or CDN link for use here would be more manageable?